desci-server/prisma/migrations/20230417045958_add_node_cover/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE "NodeCover" (
+    "id" SERIAL NOT NULL,
+    "url" TEXT NOT NULL,
+    "nodeUUID" TEXT NOT NULL,
```

Comment:
consider that cover can change even if UUID stays the same, may be best to store a CID cache and then perhaps have another table that handles UUID->CID

Reply:
ideally we don't use all-uppercase for UUID because it eventually leads to naming like

XMLHTTPAJAXRequest

instead of 

XmlHttpAjaxRequest